### PR TITLE
update: report vulnerability via github, not email

### PR DESCRIPTION
**Pull request by kikofernandez**
This PR updates how users should report security vulnerabilities in Erlang/OTP.

I have added a deprecation note on the old post from 2017, remove references to emailing us security vulnerabilities (usually users do not encrypt these messages), and copy-pasted news/111 into news/172 removing also references to bugs.erlang.org (which points to https://github.com/erlang/otp/issues).

I am going to ask @u3s and @IngelaAndin to review this description as security experts in Erlang/OTP :)

### CODE REVIEW

File: _news/172.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+The risk level is often determined by a product of the impact once exploited,
+and the probability of exploitation occurring. In other words, if a bug can
+cause great damage, but it takes highest privilege to exploit the bug, then the
+bug is not a high risk one. Similarly, if the bug is easily exploitable, but its
+impact is limited, then it is not a high risk issue either.
+
+There is not any hard and fast rule to determine if a bug is worth reporting as
+a security issue to https://github.com/erlang/otp/security. A general rule is
+that an attack by someone that has no access to the Erlang application or its
+system can affect the confidentiality, integrity and availability.
```
Comment (frej) [inline _news/172.md:29]: This sentence is incomplete. Also the standard phrase is "unprivileged user", so I would suggest:

"A general rule is that a bug which allows an unprivileged user to successfully attack the Erlang application, the Erlang runtime, or can be used as a springboard to attack other software running on the same or other machines is considered a security issue. As attacks we consider anything that affects the confidentiality, integrity and/or availability of the system."

Comment (kikofernandez) [reply _news/172.md]: solved, I think.
Thanks

**Comment by kikofernandez** [comment]
I am not sure I understand what netlify is complaining about...
I have run `make test` locally and all seems to work, and I have run `make serve` and it looks ok.

File: _news/172.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+
+## What happens after the report?
+
+All security bugs in the Erlang/OTP distribution should be reported to https://github.com/erlang/otp/security. Your report will be handled by a small security team at the OTP team.
+
+Please use a descriptive email title for your report. After the initial response to your report, the security team will keep you updated on the progress and decision being made towards a fix and release announcement.
+
+## Flagging Existing Issues as Security-related
+
```
Comment (IngelaAndin) [inline _news/172.md:32]: We should refer to GitHub secure issues here too, and get rid of the email-list in the long run I think.

Comment (kikofernandez) [reply _news/172.md:32]: solved :)

File: _news/172.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+There is not any hard and fast rule to determine if a bug is worth reporting as a security issue to [https://github.com/erlang/otp/security](https://github.com/erlang/otp/security). A general rule is that a bug which allows an unprivileged user to successfully attack the Erlang application, the Erlang runtime, or can be used as a springboard to attack other software running on the same or other machines is considered a security issue. As attacks we consider anything that affects the confidentiality, integrity and/or availability of the system.
+
+
+## What happens after the report?
+
+All security bugs in the Erlang/OTP distribution should be reported to [https://github.com/erlang/otp/security](https://github.com/erlang/otp/security). Your report will be handled by a small security team at the OTP team.
+
+Please use a descriptive email title for your report. After the initial response to your report, the security team will keep you updated on the progress and decision being made towards a fix and release announcement.
```
Comment (u3s) [inline _news/172.md:29]: what email?

File: _news/172.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+## Flagging Existing Issues as Security-related
+
+If you believe that an existing public issue on [https://github.com/erlang/otp/issues](https://github.com/erlang/otp/issues) is security-related, we ask that you report it via [https://github.com/erlang/otp/security](https://github.com/erlang/otp/security). The title should contain the issue ID on [https://github.com/erlang/otp/issues](https://github.com/erlang/otp/issues) (e.g. Flagging security issue ERL-001). Please include a short description to motivate why it should be handled according to the security policy.
```
Comment (u3s) [inline _news/172.md:33]: what is ERL-001? some leftover from old days?

**Review by u3s — CHANGES_REQUESTED** [review]
I don't know how this security tab on github really works.
I think, I don't have enough privileges for handling the items properly.

There is some procedure behind handling those items on github, I would like to understand it before we change way of working with community.

File: _news/172.md (hunk 3 above)
Comment (kikofernandez) [reply _news/172.md:29]: good catch

File: _news/172.md (hunk 4 above)
Comment (kikofernandez) [reply _news/172.md]: I took an example from `xmerl`. Please let me know if it does not look good, or if it is not a good example.
If you have any preferred way of writing this, let me know with a suggestion :)

Comment (u3s) [reply _news/172.md]: ERL-001 is a sample identifier not used anymore? is it?
you intended to refer to github issue numer - maybe GH-001 is wanted?

**Comment by garazdawi** [comment]
I pushed a fix to master for the redirect rules. If you rebase the PR, CI should now pass.

**Comment by garazdawi** [comment]
It is quite simple to add a new page with the security information instead of a news item. We could then link from it from the downloads page?

**Comment by kikofernandez** [comment]
Ok, I have added a new Security page in the header. But I can put some description in the Downloads and add it there.
I thought that the Security possibly deserves its own space to be visible everywhere to everyone.

Please, let me know if anyone disagrees, and propose change :)

**Comment by kikofernandez** [comment]
also some other remaining part regarding news/172. I think I have finished that now

File: security.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+layout: markdown
+---
+## Security
+
+Best Practice: Reporting a Security Issue in Erlang/OTP
+
+## TLDR
+Please **do not create a public issue** for a security issue, use instead [https://github.com/erlang/otp/security](https://github.com/erlang/otp/security).
+
+Please follow this document in order to report security vulnerabilities in Erlang/OTP.
```
Comment (garazdawi) [inline security.md:11]:
```suggestion
## Summary
Please create a new [Security Advisory](https://github.com/erlang/otp/security/advisories) for security issues. **Do not create a public github issue**.

Please follow this document in order to report security vulnerabilities in Erlang/OTP. 
```

Having a toddler at home has tought me that you should start with saying what you want someone to do, people always seem to miss the *not*.

**Comment by kikofernandez** [comment]
@IngelaAndin and/or @u3s , I know there were concerns about using Github Securities, at least until Security Masters feel ok using it. Whenever that happens, let me know and I merge the PR :)

**Comment by kikofernandez** [comment]
Given that we are discussing this PR soon, I also added a new file to indicate researchers how to tell us about vulnerabilities.
https://securitytxt.org/

This has been recommended by CISA, Nederland Gov, UK Gov, many Swedish websites (even H&M has one), Google, Facebook, Github, etc. It is something small that seems to serve a good purpose. I think we should add it, even if it means placing a ticket to as a reminder to renew the date.

File: .well-known/security.txt [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+Contact: https://github.com/erlang/otp/security
+
+Canonical: https://www.erlang.org/.well-known/security.txt
+
+Preferred-Languages: en
+
+# Our security policy
+Policy: https://github.com/erlang/otp/blob/master/SECURITY.md
+
+Expires: 2025-04-01T00:00:00z
```
Comment (u3s) [inline .well-known/security.txt:12]: requires a manual update every year.
supposed to be done manually or automatically on some frequent occasion.

pls update date to October/Nov?

File: security.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+Please follow this document in order to report security vulnerabilities in Erlang/OTP. 
+
+## When should you report a security issue?
+
+The risk level is often determined by a product of the impact once exploited, and the probability of exploitation occurring. In other words, if a bug can cause great damage, but it takes highest privilege to exploit the bug, then the bug is not a high risk one. Similarly, if the bug is easily exploitable, but its impact is limited, then it is not a high risk issue either.
+
+There is not any hard and fast rule to determine if a bug is worth reporting as a security issue to [https://github.com/erlang/otp/security](https://github.com/erlang/otp/security). A general rule is that a bug which allows an unprivileged user to successfully attack the Erlang application, the Erlang runtime, or can be used as a springboard to attack other software running on the same or other machines is considered a security issue. As attacks we consider anything that affects the confidentiality, integrity and/or availability of the system.
```
Comment (u3s) [inline security.md:17]: Maybe remove link ?

**Comment by kikofernandez** [comment]
Merging on Monday, 14:00 unless someone objects

**Pull request merged**